tests/v2/actions/vai/vai.go

```diff
@@ -0,0 +1,21 @@
+package vai
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/rancher/shepherd/clients/rancher"
+	"github.com/rancher/shepherd/extensions/vai"
+)
+
+func EnableVAI(client *rancher.Client, vaiEnabled *bool, once *sync.Once) error {
+	var enableErr error
+	once.Do(func() {
+		if err := vai.EnableVaiCaching(client); err != nil {
+			enableErr = fmt.Errorf("failed to enable VAI caching: %w", err)
+			return
+		}
+		*vaiEnabled = true
```

Comment:
why is this necessary? I don't think this is needed, at least not here. Is this supposed to be if viEnabled -> skip enabling it?